routeguide/protocol/src/main/resources/proto/service.proto

```diff
@@ -43,8 +43,8 @@ message RouteSummary {
 }
 
 service RouteGuideService {
-  rpc GetFeature (Point) returns (Feature);
-  rpc ListFeatures (Rectangle) returns (stream Feature);
-  rpc RecordRoute (stream Point) returns (RouteSummary);
-  rpc RouteChat (stream RouteNote) returns (stream RouteNote);
+  rpc getFeature (Point) returns (Feature);
+  rpc listFeatures (Rectangle) returns (stream Feature);
+  rpc recordRoute (stream Point) returns (RouteSummary);
```

Comment:
related to my other question: should (or is it even possible) for `Point` to be of an `observable` type?